File: domain/interfaces/iPrestamoCasoUso.go

package interfaces

import "github.com/gdosoftware/biblioteca/domain/modelo"


type  IPrestamoCasoUso interface {
    CreatePrestamo(socioId string, libroId string) (modelo.Prestamo,error)
    UpdatePrestamo(prestamoId string) (modelo.Prestamo, error)
}

File: src/main/java/com/gdosoftware/biblioteca/domain/interfaces/IPrestamoCasoUso.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.gdosoftware.biblioteca.domain.interfaces;

import com.gdosoftware.biblioteca.domain.modelo.Prestamo;

/**
 *
 * @author Dani
 */
public interface IPrestamoCasoUso {
    public Prestamo createPrestamo(Long socioId, Long libroId);
    public Prestamo devolucionPrestamo(Long prestamoId);
}
